packages/cardano-services/src/NetworkInfo/openApi.json

```
@@ -0,0 +1,179 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "Network Info Http Service",
```

```suggestion
    "title": "Network Info",
```

https://github.com/input-output-hk/cardano-js-sdk/blob/3d0e8a60e6dd2ec7820056deefbf69a37251550f/packages/cardano-services/src/ChainHistory/openApi.json#L4